apps/home/components/card/BlogCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { Image } from '@mantine/core'
+import Link from 'next/link'
+
+type Props = {
+  id: string
+  title: string,
+  author: string,
+  description: string,
+  picture: string,
+}
+
+const endpoint = process.env.NODE_ENV === 'development'
```

Comment:
Can this be used in useMemo hook?

Reply:
อันนี้เราไม่จำเป็นต้องใช้ useMemo ก็ได้ เพราะว่ามันเรียกตัวแปรนี้แค่ตอน build project ไม่ได้เรียกทุกครั้งตอนที่ component ถูกสร้าง